### PR TITLE
Acceptance tests

### PR DESCRIPTION
**Pull request by shokmaster**
This PR adds some Ember acceptance tests.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,6 +15,11 @@ env:
   global:
     - JOBS=1
 
+before_install:
```
Comment (adrigzr) [inline .travis.yml]: There is not need to move this block around.

File: .huskyrc.js [hunk 2]
```diff
@@ -3,6 +3,6 @@
 module.exports = {
 	hooks: {
 		'commit-msg': 'commitlint -E HUSKY_GIT_PARAMS',
-		'pre-push': 'npm run lint:js && npm run lint:hbs && npm test'
+		'pre-push': 'npm run lint:js && npm run && npm test'
```
Comment (adrigzr) [inline .huskyrc.js]: Restore this.

File: .travis.yml [hunk 3]
```diff
@@ -44,11 +45,22 @@ jobs:
...
-    - stage: "Regression Tests"
-      node_js: "lts/carbon"
-    - node_js: "lts/dubnium"
+    - stage: "Regression Tests Ember"
+      script:
+        - node_modules/.bin/ember try:one $EMBER_TRY_SCENARIO
+      env: EMBER_TRY_SCENARIO=ember-release
+    - env: EMBER_TRY_SCENARIO=ember-beta
+    - env: EMBER_TRY_SCENARIO=ember-canary
+    - env: EMBER_TRY_SCENARIO=ember-default-with-jquery
```
Comment (adrigzr) [inline .travis.yml:51]: This won't work as you need to repeat the script step on everyone. On the other hand, you are not cross testing all posibilities. Check this:

```
---
language: node_js
sudo: false
dist: trusty

addons:
  chrome: stable

cache:
  directories:
    - $HOME/.npm

node_js: 
  - "lts/carbon"
  - "lts/dubnium"
  - stable

env:
  - env: EMBER_TRY_SCENARIO=ember-release
  - env: EMBER_TRY_SCENARIO=ember-beta
  - env: EMBER_TRY_SCENARIO=ember-canary
  - env: EMBER_TRY_SCENARIO=ember-default-with-jquery

jobs:
  fail_fast: true
  allow_failures:
    - env: EMBER_TRY_SCENARIO=ember-canary

  include:
    - stage: "Smoke test"
      node_js: stable
      before_install:
        - npm config set spin false
        - npm install -g npm@6
        - npm install -g codecov
        - npm install -g greenkeeper-lockfile@1
        - npm install -g @commitlint/travis-cli
        - npm --version
      install:
        - if [[ $TRAVIS_BRANCH =~ greenkeeper || $TRAVIS_PULL_REQUEST_BRANCH =~ greenkeeper ]]; then npm install; else npm ci; fi
      before_script:
        - greenkeeper-lockfile-update
      script:
        - commitlint-travis
        - npm run lint:js
        - npm test
      after_script:
        - greenkeeper-lockfile-upload
      after_success:
        - codecov --file=coveragelcov.info

    - stage: "Release"
      node_js: stable
      script: skip
      deploy:
        provider: script
        skip_cleanup: true
        before_script:
          - npm uninstall husky
        script:
          - npx semantic-release

before_install:
  - npm config set spin false
  - npm install -g npm@6
  - npm --version

script:
  - node_modules/.bin/ember try:one $EMBER_TRY_SCENARIO --skip-cleanup
  - npm run test:node

stages:
  - "Smoke test"
  - test
  - name: "Release"
    if: branch = master
```

File: .watchmanconfig [hunk 4]
```diff
@@ -0,0 +1,3 @@
+{
+  "ignore_dirs": ["tmp", "dist"]
```
Comment (adrigzr) [inline .watchmanconfig:2]: What is the purpose of this file?

File: config/ember-try.js [hunk 5]
```diff
@@ -0,0 +1,58 @@
+/* eslint-env node */
+'use strict';
+
+const getChannelURL = require('ember-source-channel-url');
+
+module.exports = function() {
+	return Promise.all([
+		getChannelURL('release'),
```
Comment (adrigzr) [inline config/ember-try.js:8]: Why did you remove the LTS ones?

File: package.json [hunk 6]
```diff
@@ -10,19 +10,20 @@
...
+    "lint:js": "eslint --max-warnings 0 ./*.js config tests lib node-tests",
+    "mocha": "node_modules/.bin/mocha node-tests --recursive",
     "posttest": "npm run report-coverage && npm run check-coverage",
+    "pretest": "npm run check && npm run lint:js && npm run clean-coverage",
     "report-coverage": "istanbul report --include=coverage/coverage-final.json text",
-    "semantic-release": "semantic-release"
+    "semantic-release": "semantic-release",
+    "test": "npm run test:node && npm run test:ember",
+    "test:all": "ember try:each",
+    "test:ember": "ember test",
```
Comment (adrigzr) [inline package.json:25]: You missed `COVERAGE=true` flag. You need to add `coverage.js` file as well.

File: package.json [hunk 7]
```diff
@@ -10,19 +10,20 @@
...
+    "test:node": "nyc --reporter=html --reporter=json --reporter=json-summary --reporter=lcov --report-dir coverage --exclude node-tests --exclude **/config/* npm run mocha"
```
Comment (adrigzr) [inline package.json:26]: Change coverage destination folder and remember to merge node coverage with ember coverage.

File: package.json [hunk 8]
```diff
@@ -10,19 +10,20 @@
...
   },
```
Comment (adrigzr) [inline package.json:27]: You missed the new `validate-coverage` task 🙂 Check out the new migration script.

**Review by adrigzr — CHANGES_REQUESTED** [review]

File: .huskyrc.js (hunk 2 above)
Comment (shokmaster) [reply .huskyrc.js]: Ooops! removed alone "npm run" :+1:

File: .watchmanconfig (hunk 4 above)
Comment (shokmaster) [reply .watchmanconfig:2]: This file comes from ember-cli. I keep it for maintenance reasons.

File: package.json (hunk 6 above)
Comment (shokmaster) [reply package.json:25]: I intentionally removed it, because we don't have testeable code of Ember in this addon (only the dummy app, excluded of this purpose)

File: package.json (hunk 7 above)
Comment (shokmaster) [reply package.json:26]: replied

File: package.json (hunk 8 above)
Comment (shokmaster) [reply package.json:27]: no coverage, no validate, no problem :P

File: config/ember-try.js (hunk 5 above)
Comment (shokmaster) [reply config/ember-try.js:8]: This is exactly the same file that we've in other addons like [ember-cli-workbox](https://github.com/BBVAEngineering/ember-cli-workbox/blob/79c634ed68197ef8adb76fa3fe45a8275ffc1c0a/config/ember-try.js), and comes from  ember-cli generator.
Any way, I'm going to add 2.16 and 2.18 LTS like [ember-modal-service](https://github.com/BBVAEngineering/ember-modal-service/blob/50233bc3680670e0c7b4e1f205d9f2ec588b0ffa/config/ember-try.js), thinking about unifying criteria in the future. Thanks!

File: .travis.yml (hunk 1 above)
Comment (shokmaster) [reply .travis.yml]: restored :+1:

File: .travis.yml (hunk 3 above)
Comment (shokmaster) [reply .travis.yml:51]: Good try! But finally discarded in favour of generic .travis.yml

**Review by adrigzr — APPROVED** [review]
